tests/behavior/behavior.rs

```diff
@@ -49,9 +49,28 @@ impl BehaviorTest {
     }
 
     pub async fn run(&mut self) -> Result<()> {
-        self.test_normal().await?;
-        self.test_stat_root().await?;
-        self.test_stat_non_exist().await?;
+        let collector = {
+            let (span, collector) = Span::root("root");
+
+            let _sg1 = span.set_local_parent();
+            let mut sg2 = LocalSpan::enter_with_local_parent("a span");
+            
+            // if not using tracing macro, user can insert a keyvalue entry into span
+            sg2.add_property(|| ("a property", "a value".to_owned()));
+
+            self.test_normal().await?;
+            self.test_stat_root().await?;
+            self.test_stat_non_exist().await?;
+
+            collector
+        };
+        let spans = collector.collect().await;
+
+        // Report to Jaeger
+        let bytes =
+            minitrace_jaeger::encode("behaviour_test".to_owned(), rand::random(), 0, 0, &spans)
+                .unwrap();
+        minitrace_jaeger::report_blocking("127.0.0.1:6831".parse().unwrap(), &bytes).ok();
```

Comment:
What will happen if we don't have service at `127.0.0.1:6831`?

Reply:
`docker run --rm -d -p6831:6831/udp -p16686:16686 --name jaeger jaegertracing/all-in-one:latest`
Set up the docker at 6831, then visit `127.0.0.1:16686`  you get the pic I upload.   refer[ this](https://github.com/tikv/minitrace-rust/blob/b6ae938b3bda8771acbfccbd57bef37cc3fd87a3/minitrace-jaeger/README.md)

Reply:
> What will happen if we don't have service at `127.0.0.1:6831`?

Oh,you mean no. the collector will async collect span then push result to  `127.0.0.1:6831`, if there is no jaeger agent, it is fine. I have tested

Reply:
Thank you very much for your detailed instructions, but the question here is we should not depend on a hardcoded local address in our code (even in tests). We need to provide choices so that our users can decide whether/when/where to receive the tracing records.

I think we should only init a collector when `OPENDAL_JAEGER_ADDR` is set during test.

Reply:
> if there is no jaeger agent, it is fine. I have tested

OK, let's keep going further. We can implement the `OPENDAL_JAEGER_ADDR` related logic later. This PR is good to merge after all CI checks are passed.

Reply:
I have created an issue https://github.com/datafuselabs/opendal/issues/176 for it.

Reply:
> > Should I create anaother tests file for tracing?
> 
> We don't need to test the logic of `tracing`, they are out of the scope of `opendal`.
> 
> BTW, How about adding a new env like `OPENDAL_JAEGER_ADDR`? We can only set up a collector when we have the `OPENDAL_JAEGER_ADDR` set.

You are right,  we should leave choice to user. I'll modify it